test/admission_test.go

```diff
@@ -42,7 +47,7 @@ func TestAdmissionControl(t *testing.T) {
 			var numSuccessfulWriterTxn = atomic.Uint64{}
 			var numSuccessfulReaderTxn = atomic.Uint64{}
 
-			reader := func(first int, last int) {
+			read := func(first int, last int) {
```

Comment:
On re-reading, it's not quite clear what `first` and `last` are. Could these be made more descriptive, like `first_key_num` and `last_key_num` for example? (Not sure about these specific names)

Also, as these are both `int`s, the parameters can share the same type declaration (not sure how to describe it, I mean `first, last int`)